content/accounts.md

```diff
@@ -0,0 +1,702 @@
+---
+title: Users and Accounts
+---
+
+After reading this article, you'll know:
+
+1. What features in core Meteor enable user accounts
+1. How to use accounts-ui for a quick prototype
+1. How to use the useraccounts family of packages to build your login UI
+1. How to build a fully-featured password login experience
+1. How to enable login through OAuth providers like Facebook
+1. How to add custom data to Meteor's users collection
+1. How to manage user roles and permissions
+
+<h2 id="core-meteor">Features in core Meteor</h2>
+
+Before we get into all of the different user-facing accounts functionality you can add with Meteor, let's go over some of the features built into the Meteor DDP protocol and `accounts-base` package. These are the parts of Meteor that you'll definitely need to be aware of if you have any user accounts in your app; most of everything else is optional and added/removed via packages.
+
+<h3 id="userid-ddp">userId in DDP</h3>
+
+DDP is Meteor's built-in pub/sub and RPC protocol. You can read about how to use it in the [Data Loading](data-loading.html) and [Methods](methods.html) chapters. In addition to the concepts of data loading and remote method calls, DDP has one more feature built in - the idea of a `userId` field on a connection. This is the place where login state is tracked, regardless of which accounts UI package or login service you are using.
+
+This built-in feature means that you always get `this.userId` inside Methods and Publications, and can access the user ID on the client. This is a great starting point for building your own custom accounts system, but most developers won't need to worry about the mechanics, since you'll mostly be interacting with the `accounts-base` package instead.
+
+<h3 id="accounts-base">`accounts-base`</h3>
+
+This package is the core of Meteor's developer-facing user accounts functionality. This includes:
+
+1. A users collection with a standard schema, accessed through [`Meteor.users`](http://docs.meteor.com/#/full/meteor_users), and the client-side singletons [`Meteor.userId()`](http://docs.meteor.com/#/full/meteor_userid) and [`Meteor.user()`](http://docs.meteor.com/#/full/meteor_user), which represent the login state on the client
```

Comment:
Yep we're going to do a final edit for formatting before initial release - mostly trying to get the content correct for now